### PR TITLE
EditDoc

### LINKED ISSUE #826: Unify EditContact

### PR DESCRIPTION
**Pull request by BykhovDenis**
close #826 Signed-off-by: Denis Bykhov [80476319+BykhovDenis@users.noreply.github.com](mailto:80476319+BykhovDenis@users.noreply.github.com)

### CODE REVIEW

File: models/contact/src/index.ts [hunk 1]
```diff
@@ -124,25 +123,30 @@ export function createModel (builder: Builder): void {
...
+      component: contact.component.Contacts
+    },
+    contact.app.Contacts
+  )
 
   builder.createDoc(view.class.Viewlet, core.space.Model, {
     attachTo: contact.class.Contact,
     descriptor: view.viewlet.Table,
-    open: contact.component.EditContact,
+    open: contact.component.EditPerson,
```
Comment (haiodo) [inline models/contact/src/index.ts]: why? what about Organization?

Comment (BykhovDenis) [reply models/contact/src/index.ts]: I think we must remove this unused param....

File: plugins/view-resources/src/components/EditDoc.svelte [hunk 2]
```diff
@@ -0,0 +1,328 @@
...
+    }
+    throw new Error(`Icon not found for ${_class}`)
+  }
+
+  $: icon = object && getIcon(object._class)
+
+  let mainEditor: HTMLElement
+  let prevEditor: HTMLElement
+  let maxHeight = 0
+  const observer = new ResizeObserver(() => {
```
Comment (haiodo) [inline plugins/view-resources/src/components/EditDoc.svelte]: We discussed with @aplatoff about same size headers for all mixins per/class. So it is ok to have, but it should not be necessary.

File: models/contact/src/index.ts (hunk 1 above)
Comment (haiodo) [reply models/contact/src/index.ts]: If it is unused, let's remove it. Because it is confusing.

**Review by aplatoff — APPROVED** [review]

**Pull request merged**